lib/net/http/header.rb

```diff
@@ -207,9 +207,7 @@ def size   #:nodoc: obsolete
   # or +nil+ if there is no such key;
   # see {Fields}[rdoc-ref:Net::HTTPHeader@Fields]:
   #
-  #   res = Net::HTTP.start(hostname) do |http|
-  #     http.get('/todos/1')
-  #   end
+  #   res = Net::HTTP.get(hostname, '/todos/1')
```

Comment:
Oops, I made a mistake, `Net::HTTP.get` returns a string, we should use `Net::HTTP.HTTP.get_response` to get a `Net::HTTPResponse `.

Reply:
Et mea culpa.  I should have checked at least one complete example.